airflow/opt/providers/etna/etna/dags/decorators.py

```diff
@@ -49,19 +49,37 @@ def instantiate_dag(fn):
             start_date=start_date,
             end_date=end_date,
             schedule_interval=schedule_interval,
+            max_active_runs=1,
```

Comment:
Restricting the number of active runs so that the scheduler doesn't run multiple of overlapping instances of a single ETL.  Problem was that the genentech upload etl takes a really long time, so on the next interval it scheduled a second instance of the ETL which ... just duplicated the upload lol.   This is because the genentech etl upload uses polyphemus cursors still and not date ranges provided by airflow.  Low priority change, so I'd rather just prevent the concurrent execution.

It is possible to make multiple dag runs concurrently safe, but it is simply generally safer to keep concurrent runs of a single dag at a serial level.  We can revisit in the future.